text/0015-the-road-to-ember-2-0.md

```diff
@@ -1,5 +1,13 @@
 ---
+# FIXME: Is this correct?
+Stage: Discontinued
```

Comment:
We could also leave this as Recommended. Not quite clear how we want to handle this sort of thing.